### PR TITLE
Added forms

### PR DESCRIPTION
**Pull request by alphayowakarindi**
At this juncture of development I implemented the following:
-  Create a method called `current_user` in the ApplicationController to make current user data available to all controllers.
    -  It returns the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user` 
    - Create a comment on behalf of the `current_user` 
- Allow Users to add likes to Posts.

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -12,4 +12,8 @@ def show
     @comments_count = @post.comments.length
     @likes_count = @post.likes.length
   end
+
+  def new
+    @post = Post.new
+  end
```
Comment (CollinsTatang) [inline app/controllers/posts_controller.rb:18]: Great job so far @alphayowakarindi 

- [x] Please bring back the `create post method` inside the `posts_controller.rb` file.
backMethod

- [ ] I notice the `Method` was present but you accidentally remove it when you added the commit for `Add comments resource route`.
### **Please always test all your action method in your controllers before submitting for review**
**Happy Coding**

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @alphayowakarindi , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (alphayowakarindi) [reply app/controllers/posts_controller.rb:18]: Hi @CollinsTatang, thank you so much for taking the time to leave this excellent review. I've made the change. A lot of appreciation for your input to make my code better. Cheers!

**Review by AshakaE — APPROVED** [review]
Hi @alphayowakarindi , 

Thanks for making the requested change by the previous reviewer 👏 

### Highlights ✨ 

- Code base looks clean 👍 
- Pull request 👌 
- Git flow ✅ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Proceed to the next level 👏 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by alphayowakarindi** [comment]
Hi @AshakaE, thank you for the review. Cheers!

**Pull request merged**